hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -394,6 +394,7 @@
             '''
           resval: "1" },
         { bits: "5", name: "csb",        desc: "Direct input of CSb signal", resval: "1" },
+        { bits: "6", name: "tpm_csb",    desc: "Direct input of TPM CSb", resval: "1" }
```

Comment:
@weicaiyang I've changed the default to 1. Then `chip_csr_hw_reset` fails on this. The waveform shows the TPM CSb is 0. Could you check the chip environment if it is 0 by default? Then we should waive this from chip level test or fix the environment.

Reply:
You're right. We didn't connect this pin in chip-level. Let me see if I can quickly fix it. Or you could waive the CSR for now

Reply:
Since this signal is currently an MIO, it will be tied off to 0 at the chip-level by default.
In general, I think we waived the automatic CSR readback test for other pin readback CSRs, e.g. in `sysrst_ctrl`. we may want to do the same here:
https://github.com/lowRISC/opentitan/blob/b40e28862c7f82b149d7f233bedb211366a58fa8/hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson#L481-L483

Reply:
OK. I will waive it.

Reply:
just discussed with Michael. This is connected a dedicated IO. Before we configure the pinmux, this is default off.
Perhaps we can just waive it like we did for sysrst_ctrl
https://github.com/lowRISC/opentitan/blob/b40e28862c7f82b149d7f233bedb211366a58fa8/hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson#L481-L483